src/crewai/utilities/planning_handler.py

```diff
@@ -11,17 +12,27 @@ class PlannerTaskPydanticOutput(BaseModel):
 
 
 class CrewPlanner:
```

Comment:
I think we also need to migrate this class over to use BaseModel for validation and better typings.